### DISCUSSION TITLE
编译 wrtbwmon后在 图形界面没有相关内容

### DISCUSSION THREAD

**Discussion by bonnyandsky** [immortalwrt/immortalwrt#549] [Q&A]
无论是用仓库自带的 wrtbwmon编译
还是我删除自带拉取
svn co https://github.com/sirpdboy/sirpdboy-package/trunk/luci-app-wrtbwmon package/luci-app-wrtbwmon
svn co https://github.com/sirpdboy/sirpdboy-package/trunk/wrtbwmon package/wrtbwmon
的编译
编译成功后在系统-软件包里都能看到luci-app-wrtbwmon，luci-i18n-wrtbwmon-zh-cn已安装，但是图形界面在“服务-带宽监控”下就是没有wrtbwmon。请问是我哪里弄错了？
![image](https://user-images.githubusercontent.com/30990861/147718994-d14c93f3-d1c3-4a99-b012-20ad71670566.png)

![image](https://user-images.githubusercontent.com/30990861/147718969-5fe80ab3-d79a-4e06-bcdb-97b7e6437c58.png)

**Comment by 1715173329** [comment]
luci-app-wrtbwmon 是集成在 luci-app-nlbwmon 里面的，有个单独的“宽带监控”选项卡。
或许在 21.02 及更新版本中已经失效，考虑下使用别的插件代替。

**Reply by bonnyandsky** [reply]
> luci-app-wrtbwmon 是集成在 luci-app-nlbwmon 里面的，有个单独的“宽带监控”选项卡。 或许在 21.02 及更新版本中已经失效，考虑下使用别的插件代替。

是的，在nlbwmon （宽带监控）里面没有wrtbwmon相关的选项卡了。所以说wrtbwmon不能用了么？还是涉及到需要单独建立相关选项卡才可以？

**Comment by Pil0tXia** [comment]
通过url `luci-i18n-wrtbwmon-zh-cn` 可以访问页面，但是侧边栏中没有对应的子项
